routes/users.js

```diff
@@ -66,26 +65,27 @@ router.post("/users", async (req, res) => {
             [name, email, about, languages]
         );
 
-        res.status(201).json({ id: result.rows[0].id, message: "User created successfully." });
+        let user_id = result.rows[0].id;
+        
+        // todo add transactions: if something went wrong here, the user should not be saved
+        await userService.assignRoleToUser(user_id, 'member');
+
+        res.status(201).json({ id: user_id, message: "User created successfully." });
     } catch (err) {
         console.error(err);
         res.status(500).send("Server Error");
     }
 });
 
 // Update user by ID
-router.put("/users/:id", async (req, res) => {
+router.put("/users/:id", verifyOwnership(OWNED_ENTITIES.USER), async (req, res) => {
```

Comment:
can we change to "/users/update-user/:id"

Reply:
Discusssed in Discord channel, will keep RESTful endpoint naming